ios/XMTPReactNative.podspec

```diff
@@ -24,5 +24,5 @@ Pod::Spec.new do |s|
   }
   
   s.source_files = "**/*.{h,m,swift}"
-  s.dependency "XMTP", "= 0.1.3-beta0"
+  s.dependency "XMTP", "= 0.2.1-beta0"
```

Comment:
I'm still confused why this works, because I still don't see a 0.2.1-beta0 version for the `XMTP` cocoapod: https://github.com/search?q=repo%3ACocoaPods%2FSpecs+XMTP&type=commits

But if it works it works!